tools/modules/meterpreter_reverse.erb

```diff
@@ -9,7 +9,8 @@ module MetasploitModule
 
   include Msf::Payload::Single
   include Msf::Sessions::MeterpreterOptions
-  include Msf::Sessions::MettleConfig
+  include Msf::Sessions::MettleConfig<% unless mixins.empty? %>
+  include <%= mixins %><% end %>
```

Comment:
```suggestion
  include Msf::Sessions::MettleConfig
  <%- mixins.map do |mixin| -%>
  include <%= mixin %>
  <%- end -%>
```

Reply:
Note the explicit use of `-` here to trim whitespace, similar to kubernetes/go template syntax:

> {{- 3 }} means "trim left whitespace and print 3"
> https://helm.sh/docs/chart_template_guide/control_structures/
> https://pkg.go.dev/text/template#hdr-Text_and_spaces

As well as python's jinja:

> You can also strip whitespace in templates by hand. If you add a minus sign (-) to the start or end of a block (e.g. a [For](https://jinja.palletsprojects.com/en/3.0.x/templates/#for-loop) tag)
> https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control

For ruby's erb you need to opt into the trim mode within the erb template renderer:
```
renderer = ERB.new(template, trim_mode: '-')
```